src/sources/models.js

```diff
@@ -92,7 +91,7 @@ class Source {
   async baseUrl() {
     throw new Error("async baseUrl() must be implemented by subclasses");
   }
-  async urlFor(path, method = 'GET', headers = {}) { // eslint-disable-line no-unused-vars
+  async urlFor(path, _method = 'GET', _headers = {}) {
```

Comment:
Another downside of the `_` prefixing instead of the `no-unused-vars` disabling is that if `_method` _does_ start being used in this body, ESLint doesn't care if you rename it to remove the underscore or not. But we can detect unnecessary disablings.

Reply:
Got rid of `_`-prefixing in 8a3d795b1c63af3d073e28f2b085c5120a6f3ab1.